### PR TITLE
 - fix noevents debug flag

### PR DESCRIPTION
**Pull request by wibraun**

### CODE REVIEW

**Comment by OpenModelica-Hudson** [comment]
The test suite is unstable according to [OpenModelica_TEST_PULL_REQUEST 2015-06-24_17-33-02](https://test.openmodelica.org/hudson//job/OpenModelica_TEST_PULL_REQUEST/219/console)

**Comment by OpenModelica-Hudson** [comment]
The tests run correctly according to [OpenModelica_TEST_PULL_REQUEST 2015-06-24_17-52-07](https://test.openmodelica.org/hudson//job/OpenModelica_TEST_PULL_REQUEST/220/console)

File: Compiler/SimCode/SimCodeUtil.mo [hunk 1]
```diff
@@ -7262,9 +7262,9 @@ protected
   Integer numZeroCrossings, numTimeEvents, numRelations, numMathEventFunctions;
 algorithm
   (numZeroCrossings, numTimeEvents, numRelations, numMathEventFunctions) := BackendDAEUtil.numberOfZeroCrossings(dlow);
-  numZeroCrossings := filterNg(numZeroCrossings);
-  numTimeEvents := filterNg(numTimeEvents);
-  numRelations := filterNg(numRelations);
+  numZeroCrossings := numZeroCrossings;
```
Comment (lochel) [inline Compiler/SimCode/SimCodeUtil.mo]: What is the reason for these three lines? Why didn't you just remove them?

File: Compiler/SimCode/SimCodeUtil.mo [hunk 2]
```diff
@@ -7262,9 +7262,9 @@ protected
...
+  numTimeEvents := numTimeEvents;
+  numRelations := numRelations;
```
Comment (lochel) [inline Compiler/SimCode/SimCodeUtil.mo]: What is the reason for these three lines? Why didn't you just remove them?

**Comment by lochel** [comment]
Also the two functions [filterNg](https://github.com/OpenModelica/OMCompiler/blob/ec9f2b8e426c56b937c526b3f634c37151ac8960/Compiler/SimCode/SimCodeUtil.mo#l9262) and [useZerocrossing](https://github.com/OpenModelica/OMCompiler/blob/ec9f2b8e426c56b937c526b3f634c37151ac8960/Compiler/SimCode/SimCodeUtil.mo#l9269) are no longer used in SimCodeUtil.mo. So they should be removed as well.

**Pull request closed without merging**